app/components/avo/fields/file_field/edit_component.html.erb

```diff
@@ -1,20 +1,33 @@
 <%= field_wrapper **field_wrapper_args do %>
-  <% if @field.value.present? %>
+  <%= content_tag :div, data: {
+    controller: "file-field",
+  } do %>
     <div class="mb-2">
-      <%= render Avo::Fields::Common::Files::ViewType::GridItemComponent.new resource: @resource, field: @field %>
+      <% if @field.value.present? %>
+        <%= render Avo::Fields::Common::Files::ViewType::GridItemComponent.new resource: @resource, field: @field %>
+      <% elsif @field.is_image %>
+        <%= image_tag "", "data-file-field-target": "preview", class: "rounded-lg max-w-full" %>
+      <% elsif @field.is_audio %>
+        <%= audio_tag("", "data-file-field-target": "preview", controls: true, preload: false, class: 'w-full') %>
+      <% elsif @field.is_video %>
+        <%= video_tag("", "data-file-field-target": "preview", controls: true, preload: false, class: 'w-full') %>
```

Comment:
I don't think it's a good idea to have this duplication here.
The first time someone updates one tag here or in `GridItemComponent`, the other one will not be updated.

Can we try to use only `GridItemComponent`?